server/integration_tests/test_data/e2e_no_default_place_specified/chart_config.json

```diff
@@ -239,17 +239,6 @@
           },
           {
             "columns": [
-              {
```

Comment:
the data change has caused the best facet for these stat vars to only have a single data point: https://screenshot.googleplex.com/58S3k5urvoosXqC 

so this was demoted from TIMELINE_WITH_HIGHLIGHT to BAR_CHART & because it is only single sv and single place, highlight chart was added instead of a bar

Reply:
@ajaits This requires updating `CensusUSAMedianAgeIncome`

https://source.corp.google.com/piper///depot/google3/datacommons/import/mcf/manifest/prod_us_census_mcfs.textproto;l=95-107?q=CensusUSAMedianAgeIncome&ss=piper%2FGoogle%2FPiper:google3%2Fdatacommons%2Fimport%2Fmcf%2Fmanifest%2F

Or should we fold that into main Census (but that requires rebuilding census)?

Reply:
What exact place and variable is this? I can try it in mixer golden setup.

Reply:
https://autopush.datacommons.org/tools/timeline#statsVar=Median_Age_Person_Female&place=country/USA